Client/Frontend/Home/HomepageViewController.swift

```diff
@@ -729,10 +729,6 @@ extension HomepageViewController: HomepageContextMenuHelperDelegate {
     func homePanelDidRequestToOpenSettings(at settingsPage: Route.SettingsSection) {
         homePanelDelegate?.homePanelDidRequestToOpenSettings(at: settingsPage)
     }
-
-    func showToast(message: String) {
-        SimpleToast().showAlertWithText(message, bottomContainer: view, theme: themeManager.currentTheme)
-    }
```

Comment:
From the commit history I see that I updated the Theme for the toast but I'm not familiar with what it was used before. The only Toast I remember presented in the homepage is the undo option after delete all tabs and last tab but is not using Simple Toast because hast the Undo button but I can manually test to make sure that is still working

Reply:
Ohh do you have the test steps for that? I don't see it being used in the code, but also did run into some UI tests issues after that.

Reply:
There is also this toast:
![simulator_screenshot_E94EFC0F-C542-4E73-B2A6-2C3D2DA9ACA7](https://github.com/mozilla-mobile/firefox-ios/assets/6743397/c02647c5-192f-431d-aa80-09f703bb39e3)
ich

Reply:
For undo Close all tabs
- Open Tab tray and select the garbage can to delete all tabs 
- Tab tray will dismiss and an undo Toast should show with the option to undo

For undo close last tab:
- Open tab tray with one tab open
- Close the tab pressing in X (close button)
- Tab tray will dismiss and an undo Toast should show with the option to undo

Like I mention above I'm 99.9% sure is safe to remove this Toast because is a `SimpleToast` meaning that only displays a message and the ones I mentioned have a button so they are `ButtonToast`

Reply:
Thanks for the context! It seems the tests failing are flakey tests and @clarmso confirmed `testCloseAllTabsUndo` failing due to: https://github.com/mozilla-mobile/firefox-ios/pull/17512

I ran those tests and looks good except doesn't show up for private mode (also in production).

Reply:
`showToast` was replaced with `showShareExtension` in this PR: https://github.com/mozilla-mobile/firefox-ios/pull/17350/files